src/core/Env.js

```diff
@@ -327,6 +327,12 @@ getJasmineRequireObj().Env = function(j$) {
       }
     };
 
+    function ensureIsNotNested(method, currentSuite) {
```

Comment:
I think it's a bit confusing here to cover the outer `currentSuite` which is a function, especially if all of the callers are calling the same thing as a parameter. Additionally, we should probably be using `currentRunnable` for consistency unless there is a reason to only check for `currentSuite` instead.